src/main/java/org/example/Main.java

```diff
@@ -0,0 +1,123 @@
+        going.sendKeys("Manali");
+        driver.manage().timeouts().implicitlyWait(2,TimeUnit.SECONDS) ;
+        driver.findElement(By.xpath("//body/ul[2]/li[1]/div")).click();
+        WebElement date = driver.findElement(By.xpath("//div[3]/div[2]/div[2]/form/div[3]/input"));
+        date.click();
+        driver.manage().timeouts().implicitlyWait(2,TimeUnit.SECONDS) ;
+        set_date(driver,5);
+        driver.manage().timeouts().implicitlyWait(10,TimeUnit.SECONDS) ;
+//        submit button
+        driver.findElement(By.xpath("//div[3]/div[2]/div[2]/form/div[4]/button")).click();
+
+        driver.findElement(By.xpath("//div[3]/div[3]/div[1]/div[7]/button")).click();
+        driver.manage().timeouts().implicitlyWait(5,TimeUnit.SECONDS) ;
+//boarding point
+        driver.findElement(By.xpath("//div[@id=\"myTabContent\"]//td/input")).click();
+        //droping point
+        driver.findElement(By.xpath("//div[@id=\"myTabContent\"]//td/input")).click();
+
+
+//        seats
+
+        select_seats(driver,21,22);
+        Thread.sleep(1000);
+        driver.findElement(By.xpath("//div[@class=\"d-block text-center mt-2 mb-1\"]/button")).click();
+//        login
+        driver.findElement(By.xpath("//input[@id=\"loginuserid\"]")).sendKeys("username");
+        driver.findElement(By.xpath("//input[@id=\"pwd\"]")).sendKeys("password");
+
+        driver.findElement(By.xpath("//app-login/div[1]/form/div[3]/button")).click();
+        driver.manage().timeouts().implicitlyWait(10,TimeUnit.SECONDS) ;
+
+//        select state
+        Select drpCountry = new Select(driver.findElement(By.name("state")));
+        drpCountry.selectByVisibleText("UTTAR PRADESH");
+//    filling the Passenger  details
+        List<WebElement> name = driver.findElements(By.xpath("//input[@placeholder=\"Traveller Name\"]"));
+        name.get(0).sendKeys("Passenger x");
+        name.get(1).sendKeys("Passenger x");
+//        select gender
+        List<WebElement> gender = driver.findElements(By.name("select"));
+        Select gender1= new Select(gender.get(0));
+        gender1.selectByValue("M");
+        Select gender2= new Select(gender.get(1));
+        gender2.selectByValue("M");
+//        fill age
+        List<WebElement> age = driver.findElements(By.xpath("//input[@placeholder=\"Age\"]"));
+//
+        age.get(0).sendKeys("24");
+        age.get(1).sendKeys("24");
+
+
+
+//        I agree check box
+        driver.findElement(By.xpath("//div/form/div[3]/div[2]/div/input")).click();
+
+        driver.findElement(By.xpath("//div/form/div[3]/div[2]/button")).click();
+    }
+
+    public static void select_seats(WebDriver driver,int seat1,int seat2) throws InterruptedException {
+
+        driver.findElement(By.xpath("//div[@class=\"lowerdeck-seat-wrap\"]/span[contains(@title, \"Seat No. : "+seat1+"\")]")).click();
+
+        driver.findElement(By.xpath("//div[@class=\"lowerdeck-seat-wrap\"]/span[contains(@title, \"Seat No. : "+seat2+"\")]")).click();
```

Comment:
your locator is same for both seats, so pass only one `parm` that select seats
e.g. call method first time to select seat1 then again call to select seat 2